ui/rdb/components/SceneGraphInspectorPanel/NodeBranchView.svelte

```diff
@@ -51,10 +55,125 @@
         dispatch('open', nodeTree);
     }
 
+    let nodeInfoHideTimeout: ReturnType<typeof setTimeout>;
+    function startNodeInfoTimeout() {
+        stopNodeInfoTimeout();
+        nodeInfoHideTimeout = setTimeout(async () => {
+            nodeInfoElement.classList.add('hide');
```

Comment:
I think we ran into issues doing manual css class modifications in the past with svelte. Shouldn't this really be toggling a `.visible` property on the data, and let svelte show/hide it?

Reply:
Updated here. We do use it in the NodeDetailView and it seems to work ok but this one is a simple 1:1 relationship so I switched it